src/NzbDrone.Core/MediaFiles/DiskScanService.cs

```diff
@@ -26,6 +26,7 @@ public interface IDiskScanService
     {
         void Scan(Artist artist);
         string[] GetVideoFiles(string path, bool allDirectories = true);
```

Comment:
Yes, still need it until refactor of completed download service is complete after renaming is merged.